server/src/main/java/org/elasticsearch/threadpool/ThreadPool.java

```diff
@@ -321,6 +328,64 @@ public ThreadPool(final Settings settings, final ExecutorBuilder<?>... customBui
         this.cachedTimeThread.start();
     }
 
+    private static HashMap<String, LongGauge> setupMetrics(MeterRegistry meterRegistry, String name, ExecutorHolder holder) {
+        Map<String, Object> at = null;
+        HashMap<String, LongGauge> gauges = new HashMap<>();
+        if (holder.executor() instanceof ThreadPoolExecutor threadPoolExecutor) {
+            String prefix = "es.tp." + name + ".t."; // temporarily shortening due to max 68 chars limit
+            gauges.put(
+                "threads",
+                meterRegistry.registerLongGauge(
+                    prefix + "threads",
```

Comment:
E.g. here, would it `es.thread_pool.<name>.thread.threads`? if so, I'd suggest `es.thread_pool.<name>.thread.count`

Reply:
Yes, ok - I'll change to count instead of threads

Reply:
Fixed in fc45e177b705d1b49fff5039f1cafd714c267d97